src/main/groovy/com/auth0/gradle/oss/LibraryPlugin.groovy

```diff
@@ -210,4 +212,45 @@ class LibraryPlugin implements Plugin<Project> {
             }
         }
     }
+
+    private void japiCmp(Project project) {
+        project.afterEvaluate {
+            def lib = project.extensions.oss
+            def baselineVersion = lib.baselineVersion
+            if (baselineVersion) {
+                project.configure(project) {
+                    apply plugin: 'me.champeau.gradle.japicmp'
+                    task('apiDiff', type: JapicmpTask, dependsOn: 'jar') {
+                        oldClasspath = files(getBaselineJar(project, baselineVersion))
+                        newClasspath = files("$buildDir/libs/${project.name}-${project.version}.jar")
```

Comment:
I think you could use [archiveFile](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.Jar.html#org.gradle.api.tasks.bundling.Jar:archiveFile) here instead of hardcoding the name. Archive File is a property from the `Jar` task

Reply:
👍  new commit to use the archiveFile pushed.